src/main/resources/library/attribute-definitions.json

```diff
@@ -43,9 +45,18 @@
       "items": { "type": "string" },
       "title": "Research Institute"
     },
+    "library:diseaseOntologyDOID": {
+      "type": "string",
+      "hidden": true
+    },
+    "library:diseaseOntology": {
+      "type": "string",
+      "title": "Cohort Phenotypic Inclusion Criteria (Disease Ontology)",
+      "typeahead": "ontology"
+    },
```

Comment:
not necessary for this PR, but: at some future point we may need to indicate *which* ontology gets used for typeahead.